apps/roam/src/components/settings/NodeConfig.tsx

```diff
@@ -32,25 +41,38 @@ const NodeConfig = ({
   const overlayUid = getUid("Overlay");
   const canvasUid = getUid("Canvas");
   const graphOverviewUid = getUid("Graph Overview");
+  const githubSyncUid = getUid("GitHub Sync");
+  const githubCommentsFormatUid = getSubTree({
```

Comment:
This is probably introducing a race condition on first load because `githubSyncUid` is actually creating the node via an `async` function which `githubCommentsFormatUid` is trying to query.

I think the problem here is a bad pattern for setting up the initial configs of this setting page in general, which is out of scope for this PR.